doc/user/content/sql/show-clusters.md

```diff
@@ -38,15 +38,23 @@ default value.
 A system cluster named `mz_introspection` will be pre-installed in every
 environment. This cluster has several indexes installed to speed up common
 introspection queries, like `SHOW` commands and queries using the system
-catalog. It is recommended to switch to the `mz_introspection` cluster for
-improved performance when executing these statements.
+catalog.
+
+To take advantage of these indexes, Materialize will automatically
+re-route your `SHOW` and system catalog queries to the `mz_introspection`
+cluster. You can disable this behavior within your session via the
+`auto_route_introspection_queries`
+[session variable](/sql/show/#other-session-variables).
```

Comment:
```suggestion
To take advantage of these indexes, Materialize will automatically re-route
`SHOW` commands and queries using system catalog objects to the
[`mz_introspection` system cluster](https://materialize.com/docs/sql/show-clusters/#mz_introspection-system-cluster)
cluster. You can disable this behavior in your session via the
`auto_route_introspection_queries`[session variable](/sql/show/#other-session-variables).
```

Reply:
> [`mz_introspection` system cluster](https://materialize.com/docs/sql/show-clusters/#mz_introspection-system-cluster)

Isn't that the section we are currently in already?